README.rst

```diff
@@ -75,197 +77,186 @@ Installation
 
    cartridge --help
 
-The following commands are supported:
-
-* ``create`` — create a new application from template;
-* ``build`` — build the application for local development and testing;
-* ``start`` — start a Tarantool instance(s);
-* ``stop`` — stop a Tarantool instance(s);
-* ``status`` — get current instance(s) status;
-* ``log`` — get logs of instance(s);
-* ``clean`` - clean instance(s) files;
-* ``pack`` — pack the application into a distributable bundle;
-* ``repair`` — patch cluster configuration files;
-* `admin <doc/admin.rst>`_ - call an admin function provided by the application;
-* `replicasets <doc/replicasets.rst>`_ - manage cluster replica sets running locally;
-* `enter and connect <doc/connect.rst>`_ - connect to running instance;
-* `failover <doc/failover.rst>`_ - manage cluster failover.
-
-The following global flags are supported:
-
-* ``verbose`` — verbose mode, additional log messages are shown as well as
-  commands/docker output (such as `tarantoolctl rocks make` or `docker build` output);
-* ``debug`` — debug mode (the same as verbose, but temporary files and
-  directories aren't removed);
-* ``quiet`` — the mode that hides all logs; only errors are shown.
-
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-An application lifecycle
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Here is a list of supported Cartridge CLI commands:
+
+* ``create``: create a new application from template.
+* ``build``: build an application for local development and testing.
+* ``start``: start one or more Tarantool instances.
+* ``stop``: stop one or more Tarantool instances.
+* ``status``: get the status of one or more current instances.
+* ``log``: get logs for one or more instances.
+* ``clean``: clean files for one or more instances.
+* ``pack``: pack the application into a distributable bundle.
+* ``repair``: patch cluster configuration files.
+* `admin <doc/admin.rst>`__: call an admin function provided by the application.
+* `replicasets <doc/replicasets.rst>`__: manage cluster replica sets running locally.
+* `enter <doc/connect.rst>`__ and `connect <doc/connect.rst>`__: connect to a running instance.
+* `failover <doc/failover.rst>`_: manage cluster failover.
+
+You can control output verbosity with these global flags:
+
+* ``verbose``: displays additional log messages as well as
+  commands/docker output, such as the output of ``tarantoolctl rocks make`` or ``docker build``.
+* ``debug``: works the same as verbose, but temporary files and
+  directories aren't removed during command execution.
+* ``quiet``: hides all logs, only displays error messages.
+
+Application lifecycle
+~~~~~~~~~~~~~~~~~~~~~
 
 In a nutshell:
 
-1. `Create <Creating an application from template_>`_
-   an application (e.g. ``myapp``) from template:
+1. :ref:`Create an application <cartridge-cli-creating_an_application_from_template>`
+   (for example, ``myapp``) from a template:
```

Comment:
The link looks weird (and does not work) when rendered by GitHub. It is highly undesirable in a README file. The same for all `:ref:` and `:doc:` usages. Maybe also for other templates specific for our website. Please, re-check how the whole document looks on GitHub.

![image](https://user-images.githubusercontent.com/92749/132384538-bcca1a1a-26f4-4393-9606-7bfb69e1485e.png)

Reply:
I guess you meant 'the template' generated by the tool, not some arbitrary template.

Reply:
> I guess you meant 'the template' generated by the tool, not some arbitrary template.

At least, we don't say anything about `--dir` at this point of the document.